.github/workflows/check.yml

```diff
@@ -41,8 +41,7 @@ jobs:
       - name: Build amplify-cli
         working-directory: amplify-cli
         run: |
-          yarn policies set-version 1.18.0
-          yarn --network-concurrency 1
+          yarn dev-build
           yarn setup-dev
```

Comment:
if this references the yarn setup-dev from CLI repo, it already does the build, yarn dev-build (was removed), you might want to add `yarn` before `yarn setup-dev`